lib/kitchen/driver/exec.rb

```diff
@@ -0,0 +1,68 @@
+# -*- encoding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "kitchen/driver/base"
+require "kitchen/shell_out"
+require "kitchen/transport/exec"
+require "kitchen/version"
+
+module Kitchen
+  module Driver
+    # Simple driver that runs commands locally. As with the proxy driver, this
+    # has no isolation in general.
+    class Exec < Kitchen::Driver::Base
+      include ShellOut
+
+      plugin_version Kitchen::VERSION
+
+      default_config :reset_command, nil
+
+      no_parallel_for :create, :converge, :destroy
+
+      # Hack to inject to Exec transport in too.
```

Comment:
i'm not sure what these words mean in this order :)

Reply:
"Also inject the transport when using the driver" will change since it took me a minute to parse/remember :)